test/JDBC/expected/db_collation/test_like_index-vu-verify.out

```diff
@@ -0,0 +1,972 @@
+
+-- psql
+-- VARCHAR
+-- CS_AS
+ANALYZE master_dbo.test_like_index_vu_prepare_v_csas;
+GO
+
+-- tsql
+SET BABELFISH_SHOWPLAN_ALL ON
+GO
+
+SELECT * FROM test_like_index_vu_prepare_v_csas WHERE test_like_index_vu_prepare_v_csas_col LIKE 'cs1'
+GO
+~~START~~
+text
+Query Text: SELECT * FROM test_like_index_vu_prepare_v_csas WHERE test_like_index_vu_prepare_v_csas_col LIKE 'cs1'
+Index Only Scan using test_like_index_vu_prepare_v_cs84732f1ac470066fb5b0b1aa0dc83304 on test_like_index_vu_prepare_v_csas  (cost=0.28..8.29 rows=1 width=5)
+  Index Cond: (test_like_index_vu_prepare_v_csas_col = 'cs1'::"varchar" COLLATE bbf_unicode_cp1_cs_as)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 213.669 ms
+~~END~~
+
+
+SELECT * FROM test_like_index_vu_prepare_v_csas WHERE test_like_index_vu_prepare_v_csas_col LIKE 'cs999%'
+GO
+~~START~~
+text
+Query Text: SELECT * FROM test_like_index_vu_prepare_v_csas WHERE test_like_index_vu_prepare_v_csas_col LIKE 'cs999%'
+Bitmap Heap Scan on test_like_index_vu_prepare_v_csas  (cost=4.34..9.71 rows=1 width=5)
+  Recheck Cond: ((test_like_index_vu_prepare_v_csas_col >= 'cs999'::"varchar" COLLATE bbf_unicode_cp1_cs_as) AND (test_like_index_vu_prepare_v_csas_col < 'cs999?'::"varchar" COLLATE bbf_unicode_cp1_cs_as))
+  Filter: ((test_like_index_vu_prepare_v_csas_col)::text ~~ 'cs999%'::text COLLATE bbf_unicode_cp1_cs_as)
+  ->  Bitmap Index Scan on test_like_index_vu_prepare_v_cs84732f1ac470066fb5b0b1aa0dc83304  (cost=0.00..4.33 rows=6 width=0)
+        Index Cond: ((test_like_index_vu_prepare_v_csas_col >= 'cs999'::"varchar" COLLATE bbf_unicode_cp1_cs_as) AND (test_like_index_vu_prepare_v_csas_col < 'cs999?'::"varchar" COLLATE bbf_unicode_cp1_cs_as))
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.135 ms
+~~END~~
+
+
+SET BABELFISH_SHOWPLAN_ALL OFF
+GO
+
+SELECT * FROM test_like_index_vu_prepare_v_csas WHERE test_like_index_vu_prepare_v_csas_col LIKE 'cs1'
+GO
+~~START~~
+varchar
+cs1
+~~END~~
+
+
+SELECT * FROM test_like_index_vu_prepare_v_csas WHERE test_like_index_vu_prepare_v_csas_col LIKE 'cs999%'
+GO
+~~START~~
+varchar
+cs999
+~~END~~
+
+
+-- psql
+-- CI_AS
+ANALYZE master_dbo.test_like_index_vu_prepare_v_cias;
+GO
+
+-- tsql
+SET BABELFISH_SHOWPLAN_ALL ON
+GO
+
+SELECT * FROM test_like_index_vu_prepare_v_cias WHERE test_like_index_vu_prepare_v_cias_col LIKE 'ci1'
+GO
+~~START~~
+text
+Query Text: SELECT * FROM test_like_index_vu_prepare_v_cias WHERE test_like_index_vu_prepare_v_cias_col LIKE 'ci1'
+Index Only Scan using test_like_index_vu_prepare_v_ci2d75c4ad76ed9b58a88ba0a93bab0133 on test_like_index_vu_prepare_v_cias  (cost=0.28..8.29 rows=1 width=5)
```

Comment:
Add test where query would pick up `index scan` and not `index only scan`

Reply:
Added in latest rev